setup.cfg

```diff
@@ -91,6 +91,9 @@ xfail_strict = True
 
 [edgetest.envs.core]
 python_version = 3.9
+deps =
+    	kaleido
```

Comment:
its not python-kaleido?

Reply:
It is different between `conda` and `pip`.